src/tests/dictionaryConcepts/container/EditConcept.test.jsx

```diff
@@ -558,4 +558,10 @@ describe('Test suite for mappings on existing concepts', () => {
     instance.removeAnswerRow('uniqueKey', true, 'url', 'name', 'code', true);
     expect(spy).toHaveBeenCalled();
   });
+  it('should remove a newly created anwser row when a user is Editing a Q-A concept', () => {
+    const instance = wrapper.find('EditConcept').instance();
+    const spy = jest.spyOn(instance, 'removeAnswerRow');
```

Comment:
Are you directly making a call and then testing that it has been called?

Reply:
That is what I am doing after refactoring the test

Reply:
Can you point me to the documentation which you are following to crease such a test?

Reply:
I had a sync with the team and readjusted how I write my tests.